README.md

```diff
@@ -1,17 +1,16 @@
 ![Logo](https://github.com/user-attachments/assets/8c1534b8-6ad6-4c84-b855-2d16bf6b774a)
 
 # beamng.cluster
-BeamNG.cluster is a Node.js based server control panel for BeamMP servers. This project aims to take a simple server-manager idea and create a multi-server administration tool using an intuitive Dashboard. 
+BeamNG.cluster is a Node.js based server control panel for [BeamMP](https://github.com/beammp) servers. This project aims to take a simple server-manager idea and create a multi-server administration tool using an intuitive Dashboard. 
 
 # Media
-
-> Static prototype
+###### Static prototype
```

Comment:
From `###### Static prototype`

To `###### Prototype`

NB: Some content is no longer static, lets leave it open to interpretation of the end-user.